docs/design-proposals/2023-04-15-ebs-br-support-multiple-k8s.md

```diff
@@ -0,0 +1,242 @@
+<h2>EBS Backup Restore Federation</h2>
+
+
+<h2>Background</h2>
+
+
+Some TiDB operator users need to deploy TiDB across multiple Kubernetes clusters. Each Kubernetes cluster will be deployed in a single AZ, and each Kubernetes can only manage the AWS resources in that AZ. However, EBS backup and recovery requires the operation of volumes and snapshots in different AZs. Therefore, a federated solution needs to be provided to users to coordinate backup/restore of volumes in their respective AZs across multiple Kubernetes clusters.
+
+The key constraint is that operations involving AWS and Kubernetes resources can only be done within each Kubernetes, but TiDB components are interconnected across each Kubernetes, so TiDB-related operations are not limited by Kubernetes. Therefore, we need to categorize the EBS backup and restore process, distinguishing which processes can be done in only one Kubernetes, and which processes need to be done in each Kubernetes.
+
+<h3>Current Implementation</h3>
+
+
+In the diagrams below, the red lines represent AWS and Kubernetes related operations that need to be executed separately in each Kubernetes cluster, while the black lines represent TiDB related operations that can be executed in any Kubernetes cluster.
+
+<h4>Backup</h4>
+
+![current backup process .png](current backup process.png)
```

Comment:
it seems that the filename should have no space, otherwise, the rendered HTML is broken.

Reply:
fixed